include/engine/physic/physic.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file physic.h
+ * @author Grégory Brivady
+ * @brief Physics of the game engine
+ * @version 0.1
+ * @date 2022-12-03
+ * 
+ * 
+ */
+#ifndef ENGINE_PHYSIC_PHYSIC_H
+#define ENGINE_PHYSIC_PHYSIC_H
+
+#include <vector>
+
+#include "engine/physic/cinematic.h"
+
+/**
+ * @brief Parameters of a physic object, used to compute forces
+ * 
+ */
+struct PhxParam
+{
+    double mass = 0;
+
+    PhxParam() = default;
+} typedef PhxParam;
+
+/**
+ * @brief A Physic object, handles all the computation of the
+ * cinematic to update the position of the Entity that owns it.
+ * 
+ */
+class Physic
+{
+    friend class Entity;
+
+public:
+
+    bool is_falling = true;
+
+    PhxParam getParam();
+    Physic(/* args */) = default;
+    Physic(PhxParam phxParam){this->phxParam = phxParam;};
+
+    void setMass(double mass){this->phxParam.mass = mass;};
+    
+    /**
+     * @brief Update the position of the physic object by
+     * computing every forces that applies on it, and then
```

Comment:
"that apply to it"

Reply:
See 35c4bb85b377e074b22cde66bafe73c6e2c868e5

Reply:
:+1: